package.json

```diff
@@ -33,5 +34,13 @@
     "eslint-plugin-import": "^2.14.0",
     "eslint-plugin-jsx-a11y": "^6.1.1",
     "eslint-plugin-react": "^7.11.1"
+  },
```

Comment:
In the future, if you have something such as *configs* that is not related to the current PR. Create a new story for it and a different PR.

Reply:
Okay, I have noted that. Thanks for the insights.